Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -51,17 +46,8 @@ public bool DestroyOnSourceLost
         /// <inheritdoc />
         public virtual IMixedRealityController Controller
         {
-            get { return controller; }
-            set
-            {
-                controller = value;
-
-                if (controller != null && gameObject != null)
-                {
-                    handedness = value.ControllerHandedness;
-                    gameObject.name = $"{handedness}_{gameObject.name}";
```

Comment:
Is the gameobject name updated somewhere else now?

Reply:
Doesn't that seem weird to force it on the user? PointerName is at least assigned/controlled by the consumer.